package.json

```diff
@@ -314,6 +314,10 @@
     {
       "path": "./dist/amo-i18n-*.js",
       "maxSize": "33 kB"
+    },
+    {
+      "path": "./dist/amo-*.css",
+      "maxSize": "63 kB"
```

Comment:
I guess these are all the gzip'd sizes?  (the actual file size of the css on addons-dev seems to be 375kb, and the js files are similarly much larger)

Reply:
yep, gzipped